make/cats

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+set -o errexit -o nounset
+
+GIT_ROOT=${GIT_ROOT:-$(git rev-parse --show-toplevel)}
+NAMESPACE="cf"
+
+has_namespace() {
+    kubectl get namespace --output=name "${NAMESPACE}" >/dev/null 2>/dev/null
+}
+
+stampy "${GIT_ROOT}/scf_metrics.csv" "$0" make-cats start
+
+if has_namespace ; then
+    :
+else
+    echo 1>&2 Namespace ${NAMESPACE} is missing. SCF is not running.
+    exit 1
+fi
+
+stampy "${GIT_ROOT}/scf_metrics.csv" "$0" make-cats::setup start
+
+cf api api.cf-dev.io --skip-ssl-validation
+cf auth admin changeme
```

Comment:
The admin password is stored in `bin/settings/settings.env`, I think we could source it and use it from there.